metricbeat/module/system/process/helper.go

```diff
@@ -235,6 +241,14 @@ func (procStats *ProcStats) getProcessEvent(process *Process) common.MapStr {
 			},
 			"share": process.Mem.Share,
 		},
+		"io": common.MapStr{
+			"read_char":   process.IO.ReadChar,
```

Comment:
I suggest to map all the reads and writes together under a namespace each, means `read.bytes`, `read.count` etc ...